lib/image_optim/config.rb

```diff
@@ -157,6 +157,11 @@ def cache_worker_digests
       !!get!(:cache_worker_digests)
     end
 
+    def timeout
+      timeout = get!(:timeout)
+      timeout ? timeout.to_i : 0
```

Comment:
Hmm I usually prefer a setting to only be of a single type. Having the setting as an integer or boolean might end up causing errors in the future where integer methods are called on a boolean value.